### DISCUSSION TITLE
你也被MBTI綁架了嗎？！

### DISCUSSION THREAD

**Discussion by leotsai940914** [1101-GS4719-C/Python-Community-GS4719-C#2067] [General]
MBTI確實是和人見面閒聊的一個很好的談資，我也常常和朋友閒聊自己的MBTI，IG也追蹤了相關的粉絲專頁，每次看到各種和自己相像的INFP描述，沒有一次不跳起來驚呼「贛！這我！」，然後幫貼文點讚或轉發。

但我們卻忽略了一個根本性的問題，觀察網路上經典的16型MBTI的測驗模式，本質上就是一種分類系統，「基於你做出什麼行為來得出你的MBTI」。也就是說隨著行為、地點、時間的改變，你測得的MBTI也會不同。

可是，絕大多數人卻本末倒置，覺得因為自己是什麼MBTI，所以才會這樣，才會那樣。例如覺得自己是INFP就覺得自己「永遠」敏感與悲慘，因為這樣而「認命」不去學習改變，你才會真的永遠就只是INFP。

所以，我們要清楚認知到，它只是一種歸類，僅僅只是自己近期狀態的指標而已，如果你想改變的話隨時可以開始，加油，那不是你的命，不要忽略人有自由意志的事實。

舉我的例子，第一次測我是ESTJ，往後一段時間怎麼測都是INFP，也是我被綁架綁最深的時候，剛剛為了寫這篇我測一下是ENFP，不同時期確實會有不同狀態。但也不是說大家不要用MBTI啦，作為了解自己最近的狀態還是很有用的，也是和朋友一個很好的聊天話題。

歡迎各位分享自己最近測得的MBTI，或者你也曾經被綁架過嗎？

**Comment by leotsai940914** [comment]
順帶一提，小心不斷推送的貼文或影片，他可能會強化你「贛！山曉！這我」的認知，更有可能使你僵化思考，同溫層效應真的沒有在騙人的。

**Comment by JoannaLai909** [comment]
我覺得我其實不太會被mbti綁架欸，因為我每隔一段時間測一次都會換一種人格，應該說我會被環境影響我當下的人格，我最常在infj、isfj來回變動，這兩個人格的差別就是想法是抽象的還是實際的，所以如果我處在一個讓我ㄖ似很跳脫的環境我測出來可能就是infj，反之就又變成isfj，但這只是我個人的想法，可能每個人有自己不同的看法

**Comment by hunteryang0108** [comment]
我認為MBTI文化的發展就像星座一樣，有些人可能非常相信相關的貼文，甚至會按照貼文中的建議做決定。我認為這種情況沒有不好，如果這樣的狀況能輔助那些相對沒那麼自信的人勇敢地做出決定，增加這類人的自信心讓他勇於面對挑戰，那我認為這樣的MBTI文化也是有益於社會的。

**Comment by ds0w0** [comment]
我同意你"不要被MBTI綁架"的觀點，MBTI確實是一個很有趣的工具，用來了解自己和朋友的性格特質是個不錯的切入點。不過，我曾經看過一篇心理學雙盲測試，就是只要有部分特質符合你心裡所想的，人們就很有可能全盤接受心理測驗的結果。然而我們的性格和行為會隨著時間、環境、心境等因素而變化，所以過度依賴MBTI來定義自己或他人，可能會讓我們陷入先入為主的看法，限制了我們探索自我和成長的可能性。當我們把自己鎖在某個人格類型中，就容易忽略我們其實擁有自由意志和改變的能力。

**Comment by peresonacifer** [comment]
MBTI雖然提供了自我認識的途徑，但應該視為工具而非絕對真理，如果過度依賴這類測驗，將可能會限制個人潛能和發展，我認為重要的是保持彈性與開放心態，接受自己多面的可能性。

**Comment by william-60760** [comment]
雖然我可以理解 MBTI 能作為幫助人們更認識自己當下性格的工具，但我認為每個人的性格組成都是獨一無二的。比起 MBTI ，我更習慣時不時思考自己每個當下的反應，並將其作為自己可能會展現的性格特質紀錄下來，隨後再定期審視對自己的了解。故在這邊分享一下自己的小習慣（？），也歡迎討論有沒有其他的自我認識方法～

**Comment by Chattenoire** [comment]
建構在特質論之上的人格測試都是debatable的；庸說MBTI，就算是Big5也一樣，可以參考以下這篇論文:

Cross-culturally recurrent personality factors: Analyses of three factors (De Raad, 2005)

**Comment by CeBru13** [comment]
雖然我其實也不是很相信MBTI，但是每次聊天都會有人問我，導致我每次都要重新測試，後來我都是直接先截圖好，可以直接給別人看，我也會把他當做事一種教有的方式，用MBTI可以很輕鬆的打開話題，並且也容易找到共同的話題。

**Comment by zhilin93517** [comment]
我覺得MBTI 的概念跟占卜術很像，也很像算命師。他們的共通點似乎都是在描述對象時使用大家都符合的敘述方式進行描述，若閱讀者不跳出框架看看別的類別的敘述，很容易被蒙蔽雙眼，認為: 【欸!我真的是這樣耶】

**Comment by seastar0687** [comment]
事實上現在盛行的MBTI是由16 personalities（簡稱16p）所創造的，由可愛的小人、簡單的測驗跟好懂的分類來創造噱頭，但事實上16p的MBTI跟是延伸榮格心理類型理論，但創始人本人並沒有接受過專業的心理學與統計學的訓練及專業的榮格分析心理訓練。
而榮格的心理類型理論重點是在心理的發展動態，而不是僅僅一個固定的類型。而且並不會隨時間改變。
而16p的MBTI它通過將個體的偏好分為四對二分法來決定個人的人格類型，事實上16p的MBTI也加入了五大人格特質的理論，因此，16personalities測驗結果會更加側重於行為特徵和當下狀態，並不完全與MBTI傳統理論一致。
總的來說，其實16p的MBTI是相對有爭議的一個理論，而榮格的心理類型理論是相對更佳的心理分析

**Comment by 112121py** [comment]
MBTI這的東西，其實跟常見的血型、星座、一些心理測驗差不多，我一直以來測的都是INFP，當我看完對這個人格的描述時，覺得還蠻像我的，但也有一部份不像，所以趨於好奇心之下，我也去看了一些其他人格的描述，然後發現，裡面有些也根本是我啊xd
也許我們是可以利用這種MBTI人格分析來檢視自己的缺點，但認真想，這個世界上有那麼多人，真的能用16種人格就概括嗎

**Comment by qiu0109** [comment]
我只有測過一次MBTI，在他剛紅起來的時候我有想過可以測測看，但是發現測驗問題實在是太多就放棄，是再後來常常聽到朋友問你的MBTI是什麼才終於去測試。我覺得這就跟星座血型一樣，提供人們一個聊天話題，而且裡面的問題都是一些很明顯你選哪一種就會偏向哪一種人格，完全可以控制自己想測出來的結果，不像血型星座一樣是固定的，所以我覺得不用太在意MBTI。

**Comment by forestdew507** [comment]
有朋友分享提到，測 MBTI 時可以將自己分成兩個模式去測，一個是平常真實的自己，另一個是工作時的自己。一開始聽到覺得還蠻神奇的，後來仔細一想到底蠻有道理，畢竟一個人的樣貌還蠻多變的，並不是真的我或假的我，而是這兩個不同的時刻的確就是開啟不同的模式XD 這似乎也能和有沒有被 MBTI 綁架連結，了解不同面貌的自己而不是被一組編碼給制約:)

**Comment by ShuliANN** [comment]
我測過兩次MBTI都是一樣的結果，就會蠻容易被綁架。常常會覺得這種分類或是標籤很符合我，越是認定這些描述「就是我」，越容易自我設限，甚至不去努力改變。好像既然測出xxxx，那我就只能是這樣了，彷彿這是一個無法擺脫的標籤。

**Comment by ShuliANN** [comment]
但後來我慢慢意識到，人其實是多面的。在不同的環境、情境下，我展現出的樣貌可能完全不同。接受自己這種多變性，反而讓我感到更自由、更真實，也能更坦然地去探索未來的可能性。

**Comment by lai1008** [comment]
我其實也不是很相信MBTI，但是很常會有人聊天後問我，導致我每次都要重新測試，但是因為題目太多，所以後來我都是直接先截圖好，可以直接給別人看。同時，我也會把他當做事一種交友方式，用MBTI可以很輕鬆的打開話題，也容易找到共同的話題。

**Comment by Elaine4488** [comment]
這三年內我自己每年測了一次，三次結果都不一樣，但每一次看人格特質說明的時候又都覺得蠻像我的，而且有發現到測試的結果大多會受到當下環境的影響。

**Comment by wcyeee** [comment]
我自己覺得有些被綁架，然後還拿這個來綁架別人的比較不好一點... 平常開開玩笑說哎呀我是I我不太敢我害羞什麼的這樣我都可以接受，但有些人會一直拿出來用來當藉口。舉個例我有聽說過有人約出門被放鴿子，放鴿子的人說他是P人所以遲到爽約是正常的（不是原文）。或是要點餐問路之類的，一直叫E人出去什麼的我也是覺得這種行為不太好，綁架自己就算了綁架別人就真的是不太好。

**Comment by 113607514** [comment]
雖然我對 MBTI 沒有特別深信，但每次聊天總會有人問起，害得我常常要重新測一次。後來我乾脆直接截圖保存，方便隨時給別人看。我也把它當作一種社交工具，利用 MBTI 很容易打開話題，還能輕鬆找到彼此的共同點，增加交流的樂趣。

**Comment by kzx-creat** [comment]
以前高中測試ENFP，這一兩年測是INFP，這種測驗我覺得看看就好，因為我覺得都是統計的結果。想真正深入聊解自己還是要靠其他方法，諮商、參加不同活動、閱讀自我省思之類的。

**Comment by Taoyi-Chiang** [comment]
我覺得一堆人會用I人E人去掩飾自己是爛人的事實，但其實這本身就是不衝突的。
爛人不是因為MBTI而爛，而是因為他本身就爛。
並不是說我是I人，我就不擅長跟人互動。而是我本來就沒禮貌，剛好有個I人的特質被拿來當合理化沒禮貌的藉口而已。
